Azure/sonic-buildimage
# [as5835-54x]Set sfp port tx_disable to disable

**- Why I did it**
Set tx_disable to  disable for each boot.
Because CPLD default setting is enable. 
**- How I did it**
Add disable code to accton_as5835_54x_util.py
**- How to verify it**
Make sure tx_disable offset is  enable.
./accton_as5835_54x_util.py clean
./accton_as5835_54x_util.py install
root@sonic:/usr/local/bin# cat /sys/bus/i2c/devices/3-0061/module_tx_disable_1
0
root@sonic:/usr/local/bin# cat /sys/bus/i2c/devices/3-0061/module_tx_disable_1
0
.....
Other sfp ports tx_dis sysfs are 0.
**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
